src/assets/usage-examples-code/sprites/draw_sprite/draw_sprite-2-draw-sprite.cs

```diff
@@ -0,0 +1,37 @@
+/*
+  Usage Example: drawing a sprite
+*/
+
+using System;
+using SplashKitSDK;
+
+public class Program
```

Comment:
All C# programs on splashkit must use top level statements only, please update this, removing public class and static void

Reply:
I have made it top-level